pkg/csi/service/common/constants.go

```diff
@@ -162,17 +162,11 @@ const (
 	// IPs is Client IP address, IP range or IP subnet
 	IPs string = "ips"
 
-	// CSINamespaceVanillaK8S is the namespace of CSI driver in Vanilla K8S
-	CSINamespaceVanillaK8S = "kube-system"
+	// CNSCSINamespace is the namespace of CSI in Supervisor Cluster
+	CNSCSINamespace = "vmware-system-csi"
 
-	// CSINamespaceWorkload is the namespace of CSI in Supervisor Cluster
-	CSINamespaceWorkload = "vmware-system-csi"
-
-	// CSINamespaceTkgCluster is the namespace of CSI in TKG Cluster
-	CSINamespaceTkgCluster = "vmware-system-csi"
-
-	// CSIFeatureStatesConfigMapName is the name of configmap to store FSS values
-	CSIFeatureStatesConfigMapName = "csi-feature-states"
+	// PVCSINamespace is the namespace of CSI in TKG Cluster
+	PVCSINamespace = "vmware-system-csi"
```

Comment:
Where `PVCSINamespace` is this used in this PR?

Reply:
I had not pushed the non Vanilla k8s related code. Thanks for catching this.